tools/docker/rhel/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) .NET Foundation. All rights reserved.
+# Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM microsoft/dotnet-buildtools-prereqs:rhel-7-rpmpkg-c982313-20174116044113
+
+# Install from sudo main package TODO This package needs to be mirrored
```

Comment:
This is copied from https://github.com/dotnet/cli/blob/e3d388e82d3fb1c11d2fe0ffaaf2fc6e61464a8a/scripts/docker/rhel/Dockerfile. Not a big fan since we need to keep these files in sync. We should reduce the duplications we have between our repos.